app/main.py

```diff
@@ -1,3 +1,77 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+from app.car import fuel_price, drive_to_shop, drive_to_home
+
+
+def shop_trip() -> None:
+    with open("config.json") as file:
+        file_info = json.load(file)
+        liter_price = file_info["FUEL_PRICE"]
+        customers_info = file_info["customers"]
+        shops_info = file_info["shops"]
```

Comment:
You should declare these variables outside of the context manager; use the context manager to only retrieve the file's data to a single variable.